catalog/spatial/registry/registry-common/src/test/java/org/codice/ddf/registry/common/metacard/TestRegistryUtility.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.registry.common.metacard;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.codice.ddf.registry.common.RegistryConstants;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.impl.MetacardImpl;
+
+public class TestRegistryUtility {
+
+    private Metacard metacard;
+
+    private ArrayList<String> tags;
+
+    private Attribute tagsAttribute;
+
+    private Attribute blankRegistryIdAttribute;
+
+    private Attribute registryIdAttribute;
+
+    private Attribute identityAttribute;
+
+    private Attribute localAttribute;
+
+    @Before
+    public void setup() {
+        metacard = new MetacardImpl();
+        tags = new ArrayList<>();
+        tags.add(RegistryConstants.REGISTRY_TAG);
+        tagsAttribute = new AttributeImpl(Metacard.TAGS, tags);
+        blankRegistryIdAttribute = new AttributeImpl(RegistryObjectMetacardType.REGISTRY_ID, "");
```

Comment:
These attributes are being created before every test but aren't modified in any test. Should the be removed from the  setup?